cmd/update-main.go

```diff
@@ -111,13 +111,19 @@ func isDocker(cgroupFile string) (bool, error) {
 // is docker or not.
 func IsDocker() bool {
 	found, err := isDocker("/proc/self/cgroup")
-	fatalIf(err, "Error in docker check.")
+	// We don't need to fail for this check, print
+	// and error and return false.
+	errorIf(err, "Error in docker check.")
```

Comment:
Because we should and not fail the server. It is not binding requirement for this to be readable to start the server.